### ISSUE TITLE
Can't generate E-Mail Aliases

### ISSUE THREAD

**Issue by xDrak0** [bitwarden/android#4205]
### Production Build

- [X] I'm using the legacy Bitwarden app pubicly available in App Store / Play Store and I'm aware that Bitwarden Beta bugs should be reported in [bitwarden/android](https://github.com/bitwarden/android) or [bitwarden/ios](https://github.com/bitwarden/ios)

### Steps To Reproduce

1. Go to 'Bitwarden app'
2. Click on 'Generator > Username'
    If DDG alias isn't set up:
    a. Chose the options 'Username Type > Forwarded Email Alias' and 'Service > DuckDuckGo'
    b. Enter the API Key
3. Click on 'generate'

### Expected Result

Generate a username/alias

### Actual Result

Nothing is generated besides a error Massage:
> error sending request for url
> (https://quack.duckduckgo.com/api/email/addresses)

### Operating System

Android

### Operating System Version

13

### Build Version

2024.10.2 (19397)

**Comment by SaintPatrck** [comment]
Hi @xDrak0 

Thank you for reporting your issue.

2024.10.2 is the new native app. I've transferred your issue to the new [Android repository](https://github.com/bitwarden/android).

I'm happy to inform you that the issue has already been identified and fixed (#4197). The fix will be present in our next release.

**Comment by vuongvan** [comment]
Helllo. Im facing the same error on latest version 2024.12. How to reopen the issue.

**Comment by SaintPatrck** [comment]
Hi @zunelol 

I'm not able to reproduce the error shown in the video from the original issue. DDG alias generation is working as expected for me in the latest release. Are you seeing the same error message from your original issue, or a different one?

**Comment by SaintPatrck** [comment]
@zunelol what Android version are you using?

**Comment by SaintPatrck** [comment]
@zunelol Thank you for the additional details. I was able to replicate the issue on Android 10 and 11. I've raised this to our SDK team for investigation.

I'll keep you posted as updates are available.

**Comment by SaintPatrck** [comment]
Hi all,

Our SDK team has fixed the issue. It didn't get into this upcoming releases but should be available in the next one.

**Comment by SaintPatrck** [comment]
Fix included in [release v2025.3.0](https://github.com/bitwarden/android/releases/tag/v2025.3.0)

**Comment by uphill-papaya-shed** [comment]
Hello! I am experiencing this error in the app but not in the browser extension. Any idea what might be happening?

**Comment by uphill-papaya-shed** [comment]
> Hello! I am experiencing this error in the app but not in the browser extension. Any idea what might be happening?

![Image](https://github.com/user-attachments/assets/f42dd4d6-ccc6-485f-b420-8c66803c0026)

**Comment by tzb6js** [comment]
yes. this bug has been reintroduced  in the latest release. I have exactly the same issue on Android using the Bitwarden app. if I use it in Android Firefox with the Bitwarden extension it works OK. It also works fine on the desktop windows browser extension. it looks that the issue only relates to the standalone Google play app.
